lib/kafka_batcher/collector.ex

```diff
@@ -69,6 +70,15 @@ defmodule KafkaBatcher.Collector do
       @impl KafkaBatcher.Behaviours.Collector
       def add_events(events) do
         GenServer.call(__MODULE__, {:add_events, events})
+      catch
+        _, reason ->
+          Logger.warning(
+            "KafkaBatcher: Couldn't get through to collector",
+            reason: inspect(reason),
```

Comment:
I'm unsure about inspecting the reason and logging it. It may contain sensitive data and/or entire messages.